src/boardProvider.ts

```diff
@@ -4,6 +4,7 @@
 import { FileNames } from "./constants";
 import { Board } from "./Models/Interfaces/Board";
 import * as path from "path";
+import * as fs from "fs-plus";

```

Comment:
I recommend to use fs-extra module (which is already in package.json) instead of fs-plus

Reply:
Sure.